lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -119,9 +118,9 @@ def fixup_sti_loader!
 
             target_dir =
               case prefix
-              when core_prefix
+              when "manageiq"
                 "/var/www/miq/vmdb"
-              when /^#{gemset_prefix}/
+              when /^manageiq-gemset/
```

Comment:
minor, but once this is done, there is no longer a need to have the gemset_prefix variable, because it was only introduced to DRY things up.  Similarly target_gemset_dir can just live inside this specific `when`

Reply:
:+1: yeah that does clean things up nicely